pkg/client/client.go

```diff
@@ -1834,6 +1834,8 @@ func (c *Client) PodCapacity(ctx context.Context) (int, error) {
 // over `existing` keys and values. Additionally function performs filtering of labels and annotations from `exiting` map
 // where keys starting from string defined in `metadataPrefix` are deleted. This prevents issues with preserving stale
 // metadata defined by the operator
+//
+//nolint:errcheck
```

Comment:
I'd move the marker to the specific code lines (or go with `_ = mergo.Merge(...)`) otherwise it might mask future violations.